docs/DEPLOYMENT.md

```diff
@@ -1,14 +1,17 @@
 # Deployment
 
 - [Prerequisites](#prerequisites)
+- [Build Images](#build-images)
```

Comment:
What I think is build the image and deploy simultaneously i.e here what we are doing is building images of all db, api and ui first and then below we say edit it and deploy, instead what we can do it build the respective image and deploy in one section, wdyt ??

Reply:
all the yaml are no in root dir `config` .. if we do build and then apply, user has to keep changing directories so build all images and then apply